README.md

```diff
@@ -1,4 +1,4 @@
-# Process List Library for Go
+# Process List Library for Go [![GoDoc](https://godoc.org/github.com/mitchellh/go-ps?status.png)](https://godoc.org/github.com/mitchellh/go-ps)
```

Comment:
I think we would also need to change these urls?

Reply:
Maybe the README could be replaced completely and detail the purpose of this fork?

Reply:
Yeah that sounds like a good idea. Or just a more high up